src/test/java/httpserver/HttpPresenterTest.java

```diff
@@ -2,18 +2,48 @@
 
 import org.junit.jupiter.api.Test;
 
-import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
 class HttpPresenterTest {
     @Test
-    void itFormatsAnHttpResponse() {
+    void itFormatsAnHttpResponseWithOnlyAResponseLine() {
         HttpPresenter presenter = new HttpPresenter();
+        HttpResponse response = new HttpResponse();
+        response.responseLine += "200 Ok";
+        String responseLine = "HTTP/1.1 200 Ok" + System.lineSeparator();
+        String header = System.lineSeparator();
+        String body = "";
+        String formattedResponse = responseLine + header + body;
```

Comment:
I would probably take a similar approach here.

```
String expectedResponse = (
    "HTTP/1.1 200 Ok\r\n" +
    "\r\n";
);

assertEquals(presenter.present(response), expectedResponse);
```

Reply:
At this layer, we are working with the raw response, so I don't think we need to programmatically construct the responses in the test. Having the raw response inline makes it more clear to the user what the test is doing.

Reply:
Okay